File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/EntityAmount.java
package com.greybox.mediums.models;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EntityAmount {
    private BigDecimal agentCommissionAmount;
    private BigDecimal withholdTax;
    private BigDecimal exciseDuty;
    private BigDecimal mobileMoneyTax;
    private BigDecimal maintenanceAmountShare;
    private BigDecimal totalCharge;
    private BigDecimal netCharge;
    private BigDecimal vendorShare;
    private BigDecimal bankShare;
    private BigDecimal vendorCommission;
}

File: micropay-mobile-apis-master/Models/EntityAmount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
 
namespace micropay_apis.Models
{
	public class EntityAmount
	{
		public double commission;
		public double withHoldTax; 
		public double exciseDuty;
		public double totalCharge;
		public double netCharge;
		public double vendorShare;
		public double bankShare;
		public ResponseMessage response;
	}
}